servers/physics_3d/godot_shape_3d.h

```diff
@@ -277,6 +277,8 @@ class GodotCylinderShape3D : public GodotShape3D {
 
 struct GodotConvexPolygonShape3D : public GodotShape3D {
 	Geometry3D::MeshData mesh;
+	Vector<int> extreme_vertices;
```

Comment:
Better to always use LocalVector (it should be replaced eventually in MeshData) given its significantly faster.

Reply:
Ok, I'll fix it.  I notice LocalVector has a lot fewer methods than Vector.   It's missing `begin()` and `end()` (so it can't be iterated over) and `has()`.  Would it be ok if I add them?  If you prefer that I don't that's ok, I can work around them.